cli/src/generate/grammar_files.rs

```diff
@@ -213,6 +213,20 @@ pub fn generate_grammar_files(
                     updated = true;
                 }
 
+                // insert `tree-sitter` at the end
+                if !package_json.contains_key("tree-sitter") {
+                    eprintln!("Adding a `tree-sitter` section to package.json");
+                    package_json.insert(
+                        "tree-sitter".to_string(),
+                        json!([{
+                            "scope": format!("source.{language_name}"),
+                            "injection-regex": language_name,
+                            "file-types": [language_name],
```

Comment:
```suggestion
                            "injection-regex": format!("^{language_name}$")
```
Extensions do not always match the language name.